src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -889,6 +889,10 @@ private void WorkspaceAdded(WorkspaceModel item)
         private void WorkspaceRemoved(WorkspaceModel item)
         {
             workspaces.Remove(workspaces.First(x => x.Model == item));
+
+            // If a workspace is removed, reset this property to false 
+            // so that Run Automatic checkbox is unchecked by default
+            DynamicRunEnabled = false;
```

Comment:
@Benglin these functions are called only when we are creating a new blank workspace or creating a new workspace when opening a file. Therefore by default the `DynamicRunEnabled` flag for the new workspace being created is going to be set to `false`.